rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+Over-communication is strongly RECOMMENDED.
+
+***Every two weeks***. The Release Shepherd facilitates a one hour meeting for Product Owners, User Experience, and Technical Leads to review the progress of the Release in ZenHub and address emerging issues.
+
+The Release Shepherd MUST send a status update to the Project Leads on the DCP PM mailing list that captures action items for participants, documents slips in schedule, and/or requests clarification as needed.
+
+As required, the Release Shepherd SHOULD schedule time on the next DCP PM call to address issues pending resolution. The Project Leads MUST never be surprised by the *state of play*.
+
+***At Milestone Completion***. The Project Leads MUST update Oversight on the progress of the Release to ensure appropriate accountability.
+
+#### Modeling progress with ZenHub Pipelines [*dcp* repo]
+
+New items in the *dcp* repo appear in the ZenHub *New* pipeline and MUST be triaged into the *Product Backlog* pipeline, the *Icebox* pipeline, or the *Closed* pipeline by the Release Shepherd or Product Owners.
+
+The ZenHub *Icebox* pipeline SHOULD be used with great restraint in the *dcp* repo. Its explicit purpose is to queue up potential futures from the DCP roadmap such as pending DevSecOps issues. Individual component repo(s)  MAY use the *Icebox* pipeline in a manner of their own choosing.
+
+No issues in the *dcp* repo MUST be assigned to the *Epic* or *Sprint Backlog* pipelines. Individual component repo(s) MAY use these pipelines in a manner of their own choosing.
+
+At the level of the *dcp* repo, a subset of ZenHub pipelines are used to track progress. All Epics or issues in a Release are initially assigned to the *Product Backlog* pipeline. Once the Epic (or any of its children) is started, then its *dcp* issue is moved to the *In Progress* pipeline. When all the children of the Epic are in the *Closed* pipeline, then the *dcp* issue is moved to the *Closed* pipeline.
+
+#### Modeling progress with ZenHub Pipelines [component repo]
+
+When an issue is started, it is moved to the *In Progress* pipeline.
+
+When a pull request is filed and in review for an *In Progress* item, the issue is moved to the *Review* pipeline.
+
+If the pull request is rejected, the issue returns to *In Progress*; otherwise, it is merged and the related issue moves to the *Merged* Pipeline.
+
+When all changes are verified in [Production](https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/SOP:%20Releasing%20new%20Versions%20of%20DCP%20Software), the issue is moved to the *Closed* pipeline.
+
+### Release Demonstration and Retrospective
+
+**At Release Completion**. The Release Shepherd MUST schedule and faciliate a Release Demonstration and Retrospective meeting with the DCP Community and Oversight. It is RECOMMENDED that this occur during a DCP F2F when schedules align. 
```

Comment:
I agree that the F2F have a lot going on already. That said, if the first .5 day of the F2F was a hard look at what we all accomplished since the last meeting and what our challenges were that might not be such a bad use of our together time.

Reply:
Removed the recommendation - 4aec7a346101bbbb6e991249ea2e768b095f38f4